src/Avalonia.DotNetFrameworkRuntime/AppBuilder.cs

```diff
@@ -82,7 +82,13 @@ public AppBuilder UsePlatformDetect()
 
         private void LoadAssembliesInDirectory()
         {
-            foreach (var file in new FileInfo(Assembly.GetEntryAssembly().Location).Directory.EnumerateFiles("*.dll"))
+            var location = Assembly.GetEntryAssembly().Location;
+            if(location == null)
```

Comment:
Looking at the docs for [Assembly.Location](https://msdn.microsoft.com/en-us/library/system.reflection.assembly.location(v=vs.110).aspx) I agree that the null check is unnecessary, however it _can_ return an empty string. Maybe we should be checking for that?